### ISSUE TITLE
Ability to pass "acr_values" OpenID Connect auth request parameter

### ISSUE THREAD

**Issue by sattaman** [FormidableLabs/react-native-app-auth#30]
I am currently migrating our app from oidc-client-js and implicit flow to app auth.

A feature we are currently using is passing acr_values when logging in directly to Facebook and Google.

`oidc.signinPopup({ acr_values: 'idp:Google' })`

It looks like this feature is supported by appAuth-iOS https://github.com/openid/AppAuth-iOS/issues/199

**Comment by kadikraman** [comment]
Nice, so to confirm, the feature request is to be able to pass in anything to `additionalParameters`, in this case
```js
additionalParameters = {
  acr_values: 'idp:Google'
}
```

**Comment by sattaman** [comment]
yes please

**Comment by kadikraman** [comment]
Got a PR for this: https://github.com/FormidableLabs/react-native-app-auth/pull/33

Do you think it is ever necessary to have different `additionalParameters` for `authorize` vs `refresh`? I've made it so they're always the same at the moment, but could instead pass them directly to `authorize()` and `refresh()` functions 🤔

**Comment by sattaman** [comment]
Wow that was fast! The only thing i'd flag up is i don't always pass the same parameters. So as it stands I'd need to have a new instance of appAuth per login. I don't need to pass parameters on the refresh call, I'm not sure whether people may need to have different params there or not.

**Comment by kadikraman** [comment]
Good point. I've been thinking for a little while that the instance-based API isn't really ideal as it imposes restrictions that have no business existing. Perhaps it would be better to have a more functional API. I'll write down some ideas.

**Comment by kadikraman** [comment]
@sattaman just published`v2.0.0` and we've changed the API to be much more flexible.

So you can do e.g.

```js
const config = '<your shared config without additionalParameters>';

// authenticate
const result = await authorize({ ...config, additionalParameters: {
  acr_values: 'idp:Google'
});

// refresh
const result = await refresh(config, {
  refreshToken: result.refreshToken,
});

```
Here are the [release notes](https://github.com/FormidableLabs/react-native-app-auth/releases/tag/v2.0.0) 🚀

**Comment by sattaman** [comment]
I noticed , i refreshed you page and it had all changed , i actually checked you pr history to sanity check haha . looks good, updating now.
